README.md

```diff
@@ -1,35 +1,142 @@
+```
+
+This example:
+- Registers the host `{insert whatever the hostname is}` with Conjur, adding it into the Conjur 
+policy layer defined for the provided host factory token.
+- Installs Summon with the Summon Conjur provider for secret retrieval from Conjur.
+
+### Summon & Service Managers
+With Summon installed, using Conjur with a Service Manager (like SystemD) becomes a snap.  Here's a
+simple example of a SystemD file connecting to Conjur:
+```ini
+[Unit]
+Description=DemoApp
+After=network-online.target
+
+[Service]
+User=DemoUser
+#Environment=CONJUR_MAJOR_VERSION=4
+ExecStart=/usr/local/bin/summon --yaml 'DB_PASSWORD: !var staging/demoapp/database/password' /usr/local/bin/myapp
+```
+> Note: When connecting to Conjur 4 (Conjur Enterprise), Summon requires the environment variable
+`CONJUR_MAJOR_VERSION` set to `4`. You can provide it by uncommenting the relevant line above.
+
+The above example uses Summon to retrieve the password stored in `staging/myapp/database/password`,
+set it to an environment variable `DB_PASSWORD`, and provide it to the demo application process.
+Using Summon, the secret is kept off disk. If the service is restarted, Summon retrieves the
+password as the application is started.
 
-## Plugins
+### Recommendations
 
-### conjur_variable Lookup Plugin
+- Add `no_log: true` to each play that uses sensitive data, otherwise that data can be printed to
+  the logs.
 
-Fetch credentials from CyberArk Conjur using the controlling host's Conjur identity or environment variables.
+- Set the Ansible files to minimum permissions. Ansible uses the permissions of the user that runs
+  it.
 
-- The controlling host running Ansible has a Conjur identity. [More Information here](https://docs.conjur.org/latest/en/Content/Get%20Started/key_concepts/machine_identity.html) and here in [Conjur Ansible role project](https://github.com/cyberark/ansible-conjur-host-identity/)
-- Environment variables could be CONJUR_ACCOUNT, CONJUR_APPLIANCE_URL, CONJUR_CERT_FILE, CONJUR_AUTHN_LOGIN, CONJUR_AUTHN_API_KEY, CONJUR_AUTHN_TOKEN_FILE
+## Conjur Ansible Lookup Plugin
+Fetch credentials from CyberArk Conjur using the controlling host's Conjur identity or environment
+variables.
 
-#### Example Playbook
+he controlling host running Ansible must have a Conjur identity, provided for example by the 
+[Conjur Ansible role](#conjur-ansible-role).
+### Environment variables
+
+The following environment variables will be used by the lookup plugin to authenticate with the
+Conjur host, if they are present on the system running the lookup plugin.
+
+- `CONJUR_ACCOUNT` : The Conjur account name
+- `CONJUR_APPLIANCE_URL` : URL of the running Conjur service
+- `CONJUR_CERT_FILE` : Path to the Conjur certificate file
+- `CONJUR_AUTHN_LOGIN` : A valid Conjur host username
+- `CONJUR_AUTHN_API_KEY` : The api key that corresponds to the Conjur host username
+- `CONJUR_AUTHN_TOKEN_FILE` : Path to a file containing a valid Conjur auth token
```

Comment:
it's striking me as weird that you configure the role with playbook keys (?) and you configure the lookup plugin with env vars. how do you use them together? if I set up the role, will the plugin not be configured?

Reply:
this might be a follow-up issue (it would go hand-in-hand with your own experiments running the e2e flow)

Reply:
I think that's just the nature of how they work. The role is configured through the playbook, while the plugin is an executable that may depend on variables being present in the system. Both can be set in the playbook, but it's an important distinction.

Reply:
On the topic of "can I set them once and have them used in both places?" - I'm not sure. There's probably a way to chain things together, but that would require us to assume the user is going to always be using both the plugin and the role, and won't need to use or configure them independently. I'm not familiar with the use cases, but I assume that's something people may want to do.

Reply:
@BradleyBoutcher can we make sure it's captured in a follow-up issue that we need to review the UX of using the role and then the lookup plugin within the collection?

Reply:
#47